charts/bpndiscovery/README.md

```diff
@@ -0,0 +1,83 @@
+# bpndiscovery
```

Comment:
It would be great if we have only only true chart configuration: 
See also here https://github.com/eclipse-tractusx/sldt-bpn-discovery/blob/main/INSTALL.md

Reply:
Files adjusted